clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -186,8 +188,10 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
             credentials: credentials,
             region: region,
             asgName: serverGroupName,
-            instanceIds: instanceIds
+            instanceIds: instanceIds,
+            targetHealthyDeployPercentage: description.targetHealthyDeployPercentage ?: 100
         )
+        // @TODO pass targetHealthy% here
```

Comment:
what was the TODO here?

Reply:
It was resolved by adding targetHealthyDeployPercentage to EnableDisableAsgDescription && AbstractRegionAsgInstanceIdsDescription, removing the TODO.